pkg/virt-api/webhooks/validating-webhook/admitters/migration-create-admitter.go

```diff
@@ -36,15 +36,19 @@ import (
 
 	webhookutils "kubevirt.io/kubevirt/pkg/util/webhooks"
 	"kubevirt.io/kubevirt/pkg/virt-api/webhooks"
+	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+	"kubevirt.io/kubevirt/pkg/virt-config/featuregate"
 )
 
 type MigrationCreateAdmitter struct {
-	virtClient kubevirt.Interface
+	virtClient    kubevirt.Interface
+	ClusterConfig *virtconfig.ClusterConfig
```

Comment:
Does it need to be exported?
```suggestion
	clusterConfig *virtconfig.ClusterConfig
```

Reply:
Nope, I copied it from somewhere else and forgot to lower case it.